Sources/Fuzzilli/FuzzIL/Program.swift

```diff
@@ -27,6 +27,9 @@
 public final class Program: Collection {
     /// A program is simply a collection of instructions.
     private var instructions: [Instruction] = []
+
+    /// Runtype types of variables if available
+    public var runtimeTypes : [String : Type] = [:]
```

Comment:
yeah we can, but we need to write proto file for this structure too

Reply:
Yeah, I think making a VariableMap type in protobuf might be hard since it's a generic class. I guess we let the Program class take care of converting between the protobuf representation (either map: int -> type or just an array of types) and the VariableMap?